src/OI.cpp

```diff
@@ -22,59 +22,62 @@
 OI::OI() {
     // Process operator interface input here.
     // BEGIN AUTOGENERATED CODE, SOURCE=ROBOTBUILDER ID=CONSTRUCTORS
-    coDriver.reset(new Joystick(1));
     
-    coLGreen.reset(new JoystickButton(coDriver.get(), 15));
+	coDriver.reset(new Joystick(1));
+
+    coLGreen.reset(new JoystickButton(coDriver.get(), kLGreen_ID));
     coLGreen->WhileHeld(new AutonomousCommand());
-    coLRed.reset(new JoystickButton(coDriver.get(), 12));
+    coLRed.reset(new JoystickButton(coDriver.get(), kLRed_ID));
     coLRed->WhileHeld(new AutonomousCommand());
-    coLBlack.reset(new JoystickButton(coDriver.get(), 8));
+    coLBlack.reset(new JoystickButton(coDriver.get(), kLBlack_ID));
     coLBlack->WhileHeld(new AutonomousCommand());
-    coLWhite.reset(new JoystickButton(coDriver.get(), 13));
+    coLWhite.reset(new JoystickButton(coDriver.get(), kLWhite_ID));
     coLWhite->WhileHeld(new AutonomousCommand());
-    coLBlue.reset(new JoystickButton(coDriver.get(), 9));
+    coLBlue.reset(new JoystickButton(coDriver.get(), kLBlue_ID));
     coLBlue->WhileHeld(new AutonomousCommand());
-    coRGreen.reset(new JoystickButton(coDriver.get(), 4));
+    coRGreen.reset(new JoystickButton(coDriver.get(), kRGreen_ID));
     coRGreen->WhileHeld(new AutonomousCommand());
-    coRBlack.reset(new JoystickButton(coDriver.get(), 3));
+    coRBlack.reset(new JoystickButton(coDriver.get(), kRBlack_ID));
     coRBlack->WhileHeld(new AutonomousCommand());
-    coRRed.reset(new JoystickButton(coDriver.get(), 1));
+    coRRed.reset(new JoystickButton(coDriver.get(), kRRed_ID));
     coRRed->WhileHeld(new AutonomousCommand());
-    coRYellow.reset(new JoystickButton(coDriver.get(), 2));
+    coRYellow.reset(new JoystickButton(coDriver.get(), kRYellow_ID));
     coRYellow->WhileHeld(new AutonomousCommand());
-    coButtonStart.reset(new JoystickButton(coDriver.get(), 8));
+    coButtonStart.reset(new JoystickButton(coDriver.get(), kButtonStart_ID));
     coButtonStart->WhileHeld(new AutonomousCommand());
-    coButtonBack.reset(new JoystickButton(coDriver.get(), 7));
+    coButtonBack.reset(new JoystickButton(coDriver.get(), kButtonBack_ID));
     coButtonBack->WhileHeld(new AutonomousCommand());
-    coButtonRB.reset(new JoystickButton(coDriver.get(), 6));
+    coButtonRB.reset(new JoystickButton(coDriver.get(), kButtonRB_ID));
```

Comment:
`kButtonRB_ID`? What is the `RB`? Same problem. Avoid abbreviations.

(No change required.)